src/ucp/core/ucp_ep.c

```diff
@@ -1532,8 +1532,7 @@ static void ucp_ep_config_print_tag_proto(FILE *stream, const char *name,
     }
 
     /* print eager bcopy */
-    if (ucp_ep_is_short_lower_thresh(max_eager_short, max_bcopy) &&
-        (max_bcopy < min_rndv)) {
+    if (ucp_ep_is_short_lower_thresh(max_eager_short, max_bcopy) && max_bcopy) {
```

Comment:
what if min_rndv is less than bcopy? so that rndv starts right after the short?

Reply:
it shouldn't happen since we calculating `max_bcopy` above as follows:
```
max_bcopy = ucs_min(zcopy_thresh, min_rndv);
```